x/did/keeper/msg_server_did.go

```diff
@@ -36,10 +36,10 @@ func (m msgServer) UpdateDID(goCtx context.Context, msg *types.MsgUpdateDID) (*t
 
 	docWithSeq := keeper.GetDIDDocument(ctx, msg.DID)
 	if docWithSeq.Empty() {
-		return nil, types.Error(types.ErrDIDNotFound, msg.DID)
+		return nil, types.ErrorWrapf(types.ErrDIDNotFound, "DID: %s", msg.DID)
```

Comment:
Wrapping error.

Reply:
How about just using `sdkerrors.Wrapf()` directly? I think the `types.ErrorWrapf` has no point because it's exactly the same as `sdkerrors.Wraps()`.

Reply:
Yes. It's better way
I'll fixed it

Reply:
FIxed : https://github.com/medibloc/panacea-core/pull/129/commits/699999f03ceedb287b4d6c256946afa65f7699de